README.md

```diff
@@ -77,7 +74,28 @@ With your dependencies set up, you can run Metaphysics by running:
 yarn dev
 ```
 
-Which will start the server on http://localhost:5001
+This will pull the needed environment variables into .env.shared. If you need to override any of these values
+or add new .env values place them in the .env file.
+
+Which will start the server on http://localhost:3000
+
+You can also use hokusai by running the below command.
+
+```hokusai dev start
+
+```
+
+Be sure that memcached is no longer running before starting hokusai by running
+
+```brew services stop memcached
+
+```
+
+To test run
+
+```hokusai test
```

Comment:
There's a whole `Testing` section below in this README so I don't think this is necessary.